notebooks/ex.ipynb

```diff
@@ -0,0 +1,166 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "id": "alike-villa",
+   "metadata": {},
+   "source": [
+    "## Tell cling where to find our mfem libraries\n",
+    "\n",
+    "You may not even need this locally but for binder we do"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "administrative-dollar",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "#pragma cling add_include_path(\"/home/jovyan/local/include\")\n",
+    "#pragma cling add_library_path(\"/home/jovyan/local/lib\")"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "lovely-collins",
+   "metadata": {},
+   "source": [
+    "## Load MFEM\n",
+    "\n",
+    "Any non-default libraries must be loaded before you can `#include` files that use them"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "cultural-anthony",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "#pragma cling load(\"libmfem\")"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "explicit-vacuum",
+   "metadata": {},
+   "source": [
+    "## Now an example"
```

Comment:
Maybe show off the rich documentation of the notebook a little more with something like this:

This is the simplest MFEM example and a good starting point for new users. The example demonstrates the use of MFEM to define and solve an $H^1$ finite element discretization of the Laplace problem 
$$
\nabla^2 u = 1
$$
with homogeneous Dirichlet boundary conditions $u=0$.

The example illustrates the use of the basic MFEM classes for defining the mesh, finite element space, as well as linear and bilinear forms corresponding to the left-hand side and right-hand side of the discrete linear system.

Reply:
I agree. Feel free to just add it in and commit @rw-anderson.